ReactAndroid/src/main/java/com/facebook/react/modules/systeminfo/ReactNativeVersion.java

```diff
@@ -1,22 +1,19 @@
 /**
  * @generated by scripts/set-rn-version.js
```

Comment:
arc lint ignores files with "@generated", but the generator could also be updated to add a formatting suppression comment so that it is ignored for anything without that logic.